pkgs/development/libraries/pango/default.nix

```
@@ -27,6 +27,13 @@ in stdenv.mkDerivation rec {
       url = "https://gitlab.gnome.org/GNOME/pango/commit/fe1ee773310bac83d8e5d3c062b13a51fb5fb4ad.patch";
       sha256 = "1px66g31l2jx4baaqi4md59wlmvw0ywgspn6zr919fxl4h1kkh0h";
     })
+    # Fix issue with Pango which is breaking Pidgin
+    # See https://gitlab.gnome.org/GNOME/pango/-/issues/490
+    # Remove on next release.
+    (pkgs.fetchpatch {
```

Won't build without this
```suggestion
    (fetchpatch {
```